finary_uapi/__main__.py

```diff
@@ -56,7 +56,7 @@
     finary_uapi credit_accounts
     finary_uapi credit_accounts transactions
     finary_uapi real_estates
-    finary_uapi real_estates add rent <address> <user_estimated_value> <description> <surface> <buying_price> <building_type> <ownership_percentage> <monthly_charges> <monthly_rent> <yearly_taxes> <rental_period> <rental_type>
+    finary_uapi real_estates add rent <address> <user_estimated_value> <description> <surface> <buying_price> <building_type> <ownership_percentage> <monthly_charges> <monthly_rent> <yearly_taxes> <rental_period> <rental_type> [<currency_code>]
     finary_uapi real_estates add <category> <address> <user_estimated_value> <description> <surface> <buying_price> <building_type> <ownership_percentage>
```

Comment:
Would need to cover also the non "rent" category call also from main with optional [<currency_code>]